app/components/Content/Widget.js

```diff
@@ -25,15 +25,15 @@ const StyledCard = styled(Card).attrs({
   width: 100%;
   margin-bottom: 30px;
   position: relative;
-  padding: 24px;
+  padding: ${2 * GU}px;
 `
 
 const WidgetSelector = ({ type, data }) => {
   switch(type) {
   case TYPE_MARKDOWN:
     return <MarkdownPreview content={data.content} />
   case TYPE_VOTING:
-    return <Votes data={data} />
+    return <Voting data={data} />
```

Comment:
It does not make sense for me to see `Voting` here, that word refers to an action or verb, while `Votes` is a descriptive word and it is also the widget name and label